docker/centos-graal-maven-s2i/s2i/run

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# This POC script is (intentionally) overly trivial.
+# a TODO future PRODUCTION version of this would probably want tomake it possible to
+# e.g. pass additional parameters to the executable, as does fabric8io-images/s2i's run script.
+
+${QUARKUS_HOME}/*-runner -Dquarkus.http.host=0.0.0.0
```

Comment:
`-Dquarkus.http.host=0.0.0.0` shouldn't be needed anymore since it's now the default

Reply:
But is that already released, right now? I would prefer to keep this and then see it removed a while after a new release, together with more [generally cleaning this up in many places where it currently still appears](https://github.com/search?q=org%3Aquarkusio+quarkus.http.host&type=Code).

Reply:
The fix is in master and will be present in the `0.12.0` release that I think @cescoffier will be doing tomorrow. 
I'll leave it up to you to best decide how to handle this :)

Reply:
https://github.com/quarkusio/quarkus/pull/1312#issuecomment-474271302